intermediate_source/tiatoolbox_tutorial.rst

```diff
@@ -0,0 +1,994 @@
+Whole Slide Image Classification Using PyTorch and TIAToolbox
+=============================================================
+
+.. tip::
+   To get the most of this tutorial, we suggest using this
+   `Colab Version <https://colab.research.google.com/drive/1qkTBbMrECLjVAWwg9SJHKFgmSUyv3djE?usp=sharing>`_. This will allow you to experiment with the information presented below.
```

Comment:
Why can't we turn this into `.py` file and generate tutorial on the fly?

Reply:
Hi malfet,
We initially had it set up like this, but encountered some issues with dependency conflicts between tiatoolbox/histoencoder and the requirements of the pytorch tutorials repo. For example, tiatoolbox uses sphinx for documentation also, and there were some conflicts arising from that. And histoencoder, which is one of only a few available pathology-image-specific 'foundation models' which we use to demonstate feature extraction on whole slide images using tiatoolbox, requires a specific version of timm. Rather than trying to resolve these conflicts and potentially having the conflicts re-appearing whenever the tutorials requirements evolve, we thought it better to submit as a .rst tutorial.

Reply:
We were invited to contribute a tutorial on the use of TIAToolbox, as an example of pytorch at work in a specific application domain. TIAToolbox is a toolbox for computational pathology, so by it's nature it will be of most interest to members of that community. It is one that is growing very rapidly as we reach a critical point of available scanned tissue images, as most hospitals are moving to digital workflows.

The toolbox is quite flexible and the feature extraction, for example, could be done with any pytorch model - we could have demonstrated generating a feature map with a generic ImageNet-pretrained backbone and it would give reasonable results, but given its most likely audience is people interested in digital pathology, we thought it would be especially valuable to demonstrate how to use a pathology-specific backbone within the whole slide image processing pipelines that tiatoolbox provides.